<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SocketUi</class>
 <widget class="QDialog" name="SocketUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GPRS服务</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <widget class="QTextBrowser" name="sock_show_tb">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>451</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="sock_getip_btn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>获取ip</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sock_start_btn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>开启服务</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>152</x>
     <y>26</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>设定ip 端口:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="sock_tcp_rbtn">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>20</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TCP</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="sock_udp_rbtn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>40</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>UDP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>输入</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sock_send_btn">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>95</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sock_clear_btn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>清除记录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sock_yingyan_web_btn">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>轨迹监控</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="sock_close_btn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭服务</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sock_quit_btn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sock_ip">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>localhost:9876</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="sock_input_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>sock_quit_btn</sender>
   <signal>clicked()</signal>
   <receiver>SocketUi</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>307</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
